mmcv/transform/processing.py

```diff
@@ -0,0 +1,338 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import mmcv
+import numpy as np
+from typing import Sequence, Optional, Tuple, Union
+from .builder import TRANSFORMS
+from .base import BaseTransform
+from mmcv.image.geometric import _scale_size
+
+
+@TRANSFORMS.register_module()
+class Normalize(BaseTransform):
+    """Normalize the image.
+    Required key is "img". Added or updated keys is "img_norm_cfg",which
+    contain 'mean', 'std', 'to_rgb'.
+
+    Args:
+        mean (sequence): Mean values of 3 channels.
+        std (sequence): Std values of 3 channels.
+        to_rgb (bool): Whether to convert the image from BGR to RGB,
+            default is true.
```

Comment:
Default to True.

Reply:
Tell users the conversion from BGR to RGB is applied before or after normalization. And tell users what is the order of mean/std when to_rgb=True or False.